scratch_script.py

```diff
@@ -0,0 +1,134 @@
+# scratch script to test various implementations in datta-lab-to-nwb
+from datta_lab_to_nwb.markowitz_gillis_nature_2023 import markowitz_gillis_nature_2023behaviorinterface
+from pynwb import NWBFile
+import pandas as pd
+import numpy as np
+import matplotlib.pyplot as plt
+import colorcet as cc
+
+
+def run_conversion():
+    filename = "/Volumes/T7/CatalystNeuro/NWB/Datta/dopamine-reinforces-spontaneous-behavior/dlight_raw_data/dlight_photometry_processed_full.parquet"
+    interface = markowitz_gillis_nature_2023behaviorinterface.MarkowitzGillisNature2023BehaviorInterface(filename)
+    example_session = "2891f649-4fbd-4119-a807-b8ef507edfab"
+    metadata = interface.get_metadata(example_session)
+    metadata["reference_frame"] = "???"  # TODO: add reference frame to metadata
+    print(f"metadata: {metadata}")
+    nwbfile = NWBFile(
+        session_description=metadata["session_name"],
+        identifier=metadata["uuid"],
+        session_start_time=metadata["date"],
+        experimenter=[
+            "Jeffrey E. Markowitz",
+            "Winthrop F. Gillis",
+            "Maya Jay",
+            "Jeffrey Wood",
+            "Ryley W. Harris",
+            "Robert Cieszkowski",
+            "Rebecca Scott",
+            "David Brann",
+            "Dorothy Koveal",
+            "Tomasz Kula",
+            "Caleb Weinreb",
+            "Mohammed Abdal Monium Osman",
+            "Sandra Romero Pinto",
+            "Naoshige Uchida",
+            "Scott W. Linderman",
+            "Bernardo L. Sabatini",
+            "Sandeep Robert Datta",
+        ],
+        lab="Datta Lab",
+        institution="Harvard Medical School",
+        experiment_description="TODO",
+        session_id=metadata["uuid"],
+    )
+    nwbfile = interface.run_conversion(nwbfile, metadata, example_session)
+    return nwbfile
+
+
+def reproduce_fig1d(nwbfile):
```

Comment:
This helper is useful enough to keep around; can you move it into a utils under the source?